### DISCUSSION TITLE
Centre for any boxes

### DISCUSSION THREAD

**Discussion by JeongseopYun** [Satish-Mutyala/T24S2T1EXTG12#2] [Ideas]
Hello team, I noticed that in the home page XAML, some of the boxes are not really in the centre. Can we put everything in the middle centre? 

![image](https://github.com/user-attachments/assets/9d142011-2a8c-4471-a553-5be4ddb6045b)

**Comment by Satish-Mutyala** [comment]
Hi @Jeongseop
That's a good pick, yes we will fix the alignment issues. Thanks

**Comment by WeiZhou-dev** [comment]
Hi @JeongseopYun, thanks for bringing this up. We will fix it.
